unittests/rbus_unit_stresstest_server.cpp

```diff
@@ -840,7 +840,9 @@ TEST_F(StressTestServer, rbusMessage_GetElementsAddedByObject_test1)
 
         for(i = 0; i < num_objects; ++i)
             free(objects[i]);
-        free(objects);
```

Comment:
You can move it as you did in other place.. ie,
```
if (objects)
{
        for(i = 0; i < num_objects; ++i)
            free(objects[i]);
        free(objects);
}

Reply:
Fixed as suggested